src/role_upgrader.js

```diff
@@ -14,7 +14,7 @@ roles.upgrader.stayInRoom = true;
 roles.upgrader.buildRoad = false;
 roles.upgrader.killPrevious = true;
 
-roles.upgrader.boostActions = ['upgradeController'];
+//roles.upgrader.boostActions = ['upgradeController'];
```

Comment:
You had it disabled because too many finds so I didn't want to enable all boosting